apps/auracast.py

```diff
@@ -19,13 +19,20 @@
 import asyncio
 import contextlib
 import dataclasses
+import enum
+import functools
 import logging
 import os
 from typing import cast, Any, AsyncGenerator, Coroutine, Dict, Optional, Tuple
 
 import click
 import pyee
 
+import ctypes
+import wasmtime
```

Comment:
I would then prefer liblc3 to be a complete PyPI library that is easy to use, as we found wasm is not as useful as we expected...? I have some resource this year, so if both we and liblc3 owners agree, I can make a PyPI porting.

Reply:
I proposed https://github.com/google/liblc3/pull/61 to make it more convenient for our usage.

Reply:
I agree, a native version of liblc3 would be ideal here. `wasm` works great when we want to run in a browser page, but for a standalone python app, it just adds a heavy dependency on the wasm runtime.
The challenge then will be to also enhance the liblc3 project so that it can be managed to publish releases to PyPI with at least pre-compiled binaries for macOS, Linux and Windows.